lighthouse-core/report/v2/report-generator.js

```diff
@@ -116,7 +116,9 @@ class ReportGeneratorV2 {
    * @return {string}
    */
   generateReportHtml(reportAsJson) {
-    const sanitizedJson = JSON.stringify(reportAsJson).replace(/</g, '\\u003c');
+    const sanitizedJson = JSON.stringify(reportAsJson)
+      .replace(/</g, '\\u003c') // replaces opening script tags
+      .replace(/\u2028/g, '\\n'); // replaces line separators
```

Comment:
we also need `\u2029` (see http://timelessrepo.com/json-isnt-a-javascript-subset)

Reply:
and do we want to replace with new lines or with `\\u2028` and `\\u2029` to preserve the characters?

Reply:
yeah sure sg